Makefile

```diff
@@ -13,7 +16,13 @@ test-clj: .source-deps
 	lein with-profile +$(VERSION),+test-clj test
 
 test-cljs: .source-deps
-	lein with-profile +$(VERSION),+test-cljs test
+	if [ "$(JAVA_VERSION)" = "9.0" ] || [ "$(JAVA_VERSION)" = "9" ]; then \
```

Comment:
Also, on my linux machine, JAVA_VERSION is simply "9" instead of "9.0". Is it different on yours ?

Reply:
On Travis it returns "9.0"

Reply:
right... I was in the middle of typing the same thing.  The `orchard` builds are all getting "9.0" on the Travis CI infrastructure running Ubuntu 14.04.

What distribution are you running?

Reply:
I'm using OpenJDK on Archlinux.

Should we leave the `= "9"` around,  just in case ?

Reply:
I'm launching a Docker container with Ubuntu 16.04 and `apt-get install openjdk-9-jdk` to see. I assume that's what Travis CI will move to next.

Reply:
```
root@2735a24c8c08:~# ./lein version
Leiningen 2.8.1 on Java 9-internal OpenJDK 64-Bit Server VM
root@2735a24c8c08:~# ./lein version | cut -d " " -f 5
9-internal
root@2735a24c8c08:~# ./lein version | cut -d " " -f 5 | cut -d "." -f 1-2
9-internal
```

That `= "9"` doesn't look like it's going to work.  As of right now OpenJDK 9 on Ubuntu 16.04 calls itself `9-internal` and you could use that, but really we should open a ticket that we need a way to detect Oracle and OpenJDK versions and report back consistently. I'd be shocked if something better isn't available.

Reply:
Done.